keylime/tenant.py

```diff
@@ -615,7 +615,7 @@ def do_cv(self):
             sys.exit()
 
     def do_cvstatus(self):
-        """Perform operational state look up for agent"""
+        """Perform operational state look up for cloudverifier"""
```

Comment:
This still isn't correct. It's looking up the agent's status on the verfier. This is in contrast to the agent's status on the registrar (the other status command).

Reply:
Got you. Correcting this. "Operational state look up for agent on verifier" is a bit of a mouthful, but i'm going with this for now. I'll try to find something better for "keylimectl".